### PR TITLE
Chef 13 updates

### PR DESCRIPTION
**Pull request by ramereth**
To test, do the following on both the old and new chef-dk (Assuming you've checked this branch out on your system):
```console
$ cd ../
$ chef generate cookbook -g code_generator/ foo
$ cd foo
# This will likely fail on our older chef-dk, just remove the extra line in recipes/default.rb and try again 
# (this isn't a problem on the newer chef-dk)
$ rake
$ chef generate recipe -g ../code_generator/ bar
$ rake
$ cd ../ && rm -rf foo/
```

### CODE REVIEW

**Comment by ramereth** [comment]
[Chef 13: code_generator](https://trello.com/c/MWhb2C8V/889-chef-13-codegenerator)

**Comment by ramereth** [comment]
@TravisWhitehead I thought I had removed that file... but I must have munged it in one of my rebases accidentally. Anyway, that should be fixed now.

**Comment by ramereth** [comment]
Looks like this is having an issue on the newer chef-dk so I'll need to apply a fix for that.

**Comment by ramereth** [comment]
@TravisWhitehead try manually installing that gem into your environment and trying again. I've created a PR which should hopefully fix that specific issue.

**Comment by ramereth** [comment]
@osuosl-cookbooks/chefs could I have an additional review please?

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -11,34 +11,34 @@
...
 CENTOS_6_OPTS = {
   platform: 'centos',
   version: '6.7',
-  log_level: :fatal
+  log_level: :fatal,
 }.freeze
 
 DEBIAN_8_OPTS = {
   platform: 'debian',
   version: '8.4',
```
Comment (eldebrim) [inline spec/spec_helper.rb]: Should this and `CENTOS_6_OPTS` be updated as well?

**Pull request merged**